src/main/js/helpers/styles.js

```diff
@@ -111,6 +111,8 @@ export default {
     activityGroup: "carbon-data-activity-group",
     activity: "carbon-data-activity",
     activityBar: "carbon-data-activity-bar",
+    eventGroup: "carbon-data-event-group",
+    actionGroup: "carbon-data-event-group",
```

Comment:
Should this be `carbon-data-action-group`?

Reply:
I added it for some functionality but ended up not using them. So, Removed it here https://github.com/cerner/carbon-graphs/pull/138/commits/2da68b4daa7cd2ee30db5f7473727625d7347e55.